src/package/components/Tabs/index.js

```diff
@@ -1,8 +1,8 @@
 import React, {Component} from 'react'
-import {Tab, Tabs, TabList, TabPanel} from 'react-tabs'
+import {Tab, Tabs, TabList, TabPanel, resetIdCounter} from 'react-tabs'
 import {map} from 'lodash'
 
-Tabs.setUseDefaultStyles(false)
+resetIdCounter()
```

Comment:
Since the react-tabs 1.0.0 release upgraded to in this PR, default styles are no longer applied and server-rendering is supported with the `resetIdCounter` so the server and client renders have the same checksum